pkg/executor/executortype/poolmgr/gp_deployment.go

```diff
@@ -34,7 +35,18 @@ import (
 // getPoolName returns a unique name of an environment
 func (gp *GenericPool) getPoolName(env *fv1.Environment) string {
```

Comment:
Lets convert this function from method to simple pure function, ie,

```
func getPoolName(env *fv1.Environment) string {
...
}
```
We should unit tests around this function.